### PR TITLE
Complete webpack reconfiguration, major changes

### PR DESCRIPTION
**Pull request by serhii-lypko**

### CODE REVIEW

**Review by kulyk — APPROVED** [review]
LGTM

File: src/components/Widget/index.tsx [hunk 1]
```diff
@@ -71,15 +76,17 @@ function Widget(props: Props) {
...
-
-      dispatch(setDialogConfig(dialogConfig));
-      dispatch(setWidgetParameters(widgetParameters));
-      dispatch(setDialogActiveMessage(firstStep));
-      dispatch(addResponseMessage(firstStep.message));
-    });
+    Promise.all([fetchDialogScript, fetchWidgetLookConfig]).then(
+      ([script, { widgetParameters }]) => {
+        const dialogConfig = script as DialogConfig;
+        const firstStep = dialogConfig.script[dialogConfig.firstStepId];
```
Comment (dmyur) [inline src/components/Widget/index.tsx:82]: 🙌

File: src/index.tsx [hunk 2]
```diff
@@ -1,59 +1,15 @@
-import React from 'react';
-import { Provider } from 'react-redux';
+import React from "react";
+import { render, unmountComponentAtNode } from "react-dom";
+import App from "./App";
 
-import Widget from './components/Widget';
+const MOUNT_NODE = document.getElementById("root");
```
Comment (dmyur) [inline src/index.tsx]: Let's make it something like `dancecard-leadnurturing-chatbot` so that we don't forget.

File: src/store/types.ts [hunk 3]
```diff
@@ -53,7 +53,7 @@ export type Testimonial = {
 };
 
 export interface DialogConfig {
-  firstStepId: string | undefined;
+  firstStepId: string;
```
Comment (dmyur) [inline src/store/types.ts:56]: ✅

**Review by dmyur — APPROVED** [review]
Let's roll.

**Pull request merged**